here_for_change/municipalities/templates/municipalities/ward_detail.html

```diff
@@ -159,7 +159,7 @@ <h2 class="text-2xl md:text-3xl benfritz text-white text-center">
               <i class="fa-solid fa-envelope"></i>
             </span>
             <span class="text-sm mr-1 md:mr-5 hover:bg-gray-200 p-2 rounded-md cursor-pointer whitespace-nowrap">
-              {{content.councillor.contacts.email.primary.value}}
+              {{content.ward.councillor.contacts.email.primary.value}}
```

Comment:
@Hazmid note that the data has moved up a level so it was `content.councillor.contacts.email.primary.value` but is now `content.ward.councillor.contacts.email.primary.value`. This allows for `content.municipality.office_of_the_deputy_mayor.secretary_deputy_mayor.name.value`

Reply:
okay, noted.